src/draw/drawMarker.ts

```diff
@@ -46,20 +45,21 @@ export function drawMarker(
 ): Image {
   const newImage = getOutputImage(image, options, { clone: true });
   const {
-    color = getDefaultColor(newImage),
-    filled = false,
+    strokeColor = getDefaultColor(newImage),
+    fillColor = getDefaultColor(newImage),
```

Comment:
Doesn't this make the marker filled by default when it wasn't before?

How do you not fill the marker? (here passing `undefined` explicitly will still assign a default color).

Reply:
Fixed.